pkg/service/system/route53.go

```diff
@@ -129,3 +182,179 @@ func AddRoute53Record(ctx context.Context, dnsName, recordName, regionName strin
 
 	return *result.ChangeInfo.Id, nil
 }
+
+// GetRoute53TXTRecords will return TXT records only
+func GetRoute53TXTRecords(ctx context.Context) ([]types.Route53Record, error) {
+
+	hostedZoneId := config.Get().AwsRoute53HostedZoneID
+
+	// Creating AWS Route53 session
+	route53Client, err := getRoute53Sess(emptyRegion)
+	if err != nil {
+		return nil, errors.Wrap(err, "GetRoute53Records: failed to create route53 session")
+	}
+
+	getRecordsInput := &route53.ListResourceRecordSetsInput{
+		HostedZoneId: aws.String(hostedZoneId),
+		MaxItems:     aws.String("1000"),
```

Comment:
Why aren't we also adding filter for `RecordType` `TXT`?

Reply:
There is no way to filter the records with request, filtering the records later.

Reply:
`StartRecordType: aws.String("TXT")` will filter for record type